sass-graph.js

```diff
@@ -63,20 +67,20 @@ Graph.prototype.addFile = function(filepath, parent) {
 
   var i, length = imports.length, loadPaths, resolved;
   for (i = 0; i < length; i++) {
-    loadPaths = _([cwd, this.dir]).concat(this.loadPaths).filter().uniq().value();
```

Comment:
The chained syntax requires that the whole `lodash` package is available. I wouldn't recommend it in regular projects but it I think it is not at all advisable in libraries.